QuestionBot/QuestionBot/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Castle.Core" version="3.3.3" targetFramework="net452" />
+  <package id="NUnit" version="3.4.1" targetFramework="net452" />
```

Comment:
How come NUnit is being installed in the project? it should only be in the test project.

Don't worry about this for now, we can deal with it in another PR :)